cohort-engine/src/main/java/com/ibm/cohort/engine/measure/ResourceResolutionProvider.java

```diff
@@ -286,12 +292,7 @@ public Measure resolveMeasureByIdentifier(Identifier identifier, String version)
 				} else {
 					Optional<SemanticVersion> candidateVersion = SemanticVersion.create(measure.getVersion());
 					if (candidateVersion.isPresent()) {
-						if (latestVersion == null) {
```

Comment:
> Change this condition so that it does not always evaluate to "true".

Reading the code, latestVersion is never assigned, and therefore always true.  Since we're never hitting the else, I removed it.  If this code should have alternative behavior please speak up.

Reply:
@csandersdev would you be able to sanity check  this?

The code I originally wrote for  semantic version cared about finding the latest semantic version when no version was specified. This change makes it seem that whatever entry is latest in `resourcesByIdByResourceType.get(MEASURE).values()` would just win, regardless of version.

Reply:
@msargentibm  You are right. I introduced a bug when I brought the code over. It should set the latestVersion = candidateVersion if latestVersion is null and then, later, only update the latestVersion if it is newer than the previous latestVersion. I will pull this branch and fix.